Marlin/src/feature/runout.h

```diff
@@ -334,8 +369,10 @@ class FilamentSensorBase {
         #endif
       }
 
-      static inline bool has_run_out() {
-        return runout_mm_countdown[active_extruder] < 0;
+      static inline uint8_t has_run_out() {
+        uint8_t sensor_bitmask = 0;
+        LOOP_L_N(i, NUM_RUNOUT_SENSORS) if (runout_mm_countdown[i] < 0) SBI(sensor_bitmask, i);
```

Comment:
Also seems to ignore whether your feature is enabled?

Reply:
This applies to everything below this in the file also.

Reply:
The state is checked for all sensors, however a runout is only triggered for the active extruder / sensor or when WATCH_ALL_RUNOUT_SENSORS is enabled.